Tests/Helper/FragmentHelperTest.php

```diff
@@ -39,11 +39,12 @@ protected function setUp()
         $this->fragmentHelper = new FragmentHelper($this->templating);
     }
 
+    /**
+     * @expectedException \RuntimeException
+     * @expectedExceptionMessage Cannot render Fragment of type `foo.bar`. Service not found.
+     */
     public function testRenderWithoutService()
     {
-        $this->expectException('\RuntimeException');
-        $this->expectExceptionMessage('Cannot render Fragment of type `foo.bar`. Service not found.');
```

Comment:
Please use `setExpectedException` with multiple arguments instead of annotations.